### PR TITLE
[List] Add TypographyThemer to MDCSelfSizingStereoCell

### LINKED ISSUE #5297: [List] Add TypographyThemer to MDCSelfSizingStereoCell
Add a typography themer to MDCSelfSizingStereoCell.

### PR DESCRIPTION
**Pull request by andrewoverton**
Closes #5297

### CODE REVIEW

File: components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+@interface MDCSelfSizingStereoCellTypographyThemer : NSObject
+
+/**
+ Applies a typography scheme's properties to an MDCSelfSizingStereoCell.
+
+ @param typographyScheme The typography scheme to apply to the component instance.
+ @param cell A component instance to which the typography scheme should be applied.
+ */
++ (void)applyTypographyScheme:(id<MDCTypographyScheming>)typographyScheme
+      toActionSheetController:(MDCSelfSizingStereoCell *)cell;
```
Comment (codeman7) [inline components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.h]: _tip:_ Rename from ActionSheetController to BaseCell

File: components/List/examples/MDCSelfSizingStereoCellExample.m [hunk 2]
```diff
@@ -113,6 +117,8 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
   cell.leadingImageView.tintColor = [UIColor darkGrayColor];
   cell.trailingImageView.tintColor = [UIColor darkGrayColor];
   cell.mdc_adjustsFontForContentSizeCategory = YES;
+  [MDCSelfSizingStereoCellTypographyThemer applyTypographyScheme:self.typographyScheme
```
Comment (codeman7) [inline components/List/examples/MDCSelfSizingStereoCellExample.m:120]: _tip:_ Update after the API change.

File: components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.m [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCSelfSizingStereoCellTypographyThemer.h"
+
+@implementation MDCSelfSizingStereoCellTypographyThemer
+
++ (void)applyTypographyScheme:(id<MDCTypographyScheming>)typographyScheme
+      toActionSheetController:(MDCSelfSizingStereoCell *)cell {
+  cell.titleLabel.font = typographyScheme.subtitle1;
```
Comment (codeman7) [inline components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.m:21]: _tip:_ Again change the API

**Review by codeman7 — DISMISSED** [review]

File: components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.h (hunk 1 above)
Comment (andrewoverton) [reply components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.h]: LOL whoops

File: components/List/examples/MDCSelfSizingStereoCellExample.m (hunk 2 above)
Comment (andrewoverton) [reply components/List/examples/MDCSelfSizingStereoCellExample.m:120]: done.

File: components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.m (hunk 3 above)
Comment (andrewoverton) [reply components/List/src/TypographyThemer/MDCSelfSizingStereoCellTypographyThemer.m:21]: done.

**Review by codeman7 — APPROVED** [review]

**Review by ianegordon — APPROVED** [review]

**Pull request merged**